homeassistant/components/fitbit/exceptions.py

```diff
@@ -0,0 +1,12 @@
+"""Exceptions for fitbit API calls.
+
+These exceptions exist to provide common exceptions for the async and sync client libraries.
+"""
+
+
+class FitbitApiException(Exception):
```

Comment:
Should we inherit from `HomeAssistantError`?

Reply:
While updating this, I also added a `reason` to each which can be used in the config flow as the abort reason. Not sure its a big win, but seemed useful.